<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingViewWidget</class>
 <widget class="QWidget" name="DrawingViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="drawingNumberHeaderLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: rgb(70, 70, 70);
}</string>
     </property>
     <property name="text">
      <string>DRAWING NUMBER</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="drawingViewTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="drawingSpecsTab">
      <attribute name="title">
       <string>Drawing Specs</string>
      </attribute>
      <layout class="QFormLayout" name="drawingSpecsTabFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="drawingNumberLabel">
         <property name="text">
          <string>Drawing Number:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="drawingNumberTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="productLabel">
         <property name="text">
          <string>Product:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lengthLabel">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="apertureLabel">
         <property name="text">
          <string>Aperture:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="sidelapsLabel">
         <property name="text">
          <string>Sidelaps:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="overlapsLabel">
         <property name="text">
          <string>Overlaps:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="sideIronsLabel">
         <property name="text">
          <string>Side Irons:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="thicknessLabel">
         <property name="text">
          <string>Thickness:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="materialsLabel">
         <property name="text">
          <string>Materials:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>Notes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="productTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="widthTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lengthTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="apertureTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="sidelapsTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="overlapsTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="leftSideIronTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="thicknessTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="topMaterialTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QTextEdit" name="notesTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="rightSideIronTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="bottomMaterialTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="machineTemplateTab">
      <attribute name="title">
       <string>Machine Template</string>
      </attribute>
      <layout class="QFormLayout" name="machineTemplateTabFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="machineLabel">
         <property name="text">
          <string>Machine:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="machineTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="quantityOnDeckLabel">
         <property name="text">
          <string>Quantity On Deck:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="quantityOnDeckTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="positionTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="deckLabel">
         <property name="text">
          <string>Deck:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="deckTextbox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="drawingPDFsTab">
      <attribute name="title">
       <string>Drawings</string>
      </attribute>
      <layout class="QFormLayout" name="drawingPDFsTabFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="drawingPDFSelectorLabel">
         <property name="text">
          <string>Drawing:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="drawingPDFSelectorInput"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>drawingViewTabs</tabstop>
  <tabstop>drawingNumberTextbox</tabstop>
  <tabstop>productTextbox</tabstop>
  <tabstop>widthTextbox</tabstop>
  <tabstop>lengthTextbox</tabstop>
  <tabstop>apertureTextbox</tabstop>
  <tabstop>sidelapsTextbox</tabstop>
  <tabstop>overlapsTextbox</tabstop>
  <tabstop>leftSideIronTextbox</tabstop>
  <tabstop>rightSideIronTextbox</tabstop>
  <tabstop>thicknessTextbox</tabstop>
  <tabstop>topMaterialTextbox</tabstop>
  <tabstop>bottomMaterialTextbox</tabstop>
  <tabstop>notesTextbox</tabstop>
  <tabstop>machineTextbox</tabstop>
  <tabstop>quantityOnDeckTextbox</tabstop>
  <tabstop>positionTextbox</tabstop>
  <tabstop>deckTextbox</tabstop>
  <tabstop>drawingPDFSelectorInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
